lldb/tools/lldb-dap/DAP.h

```diff
@@ -123,21 +126,21 @@ struct Variables {
 
 struct StartDebuggingRequestHandler : public lldb::SBCommandPluginInterface {
   DAP &dap;
-  explicit StartDebuggingRequestHandler(DAP &d) : dap(d) {};
+  explicit StartDebuggingRequestHandler(DAP &d) : dap(d){};
```

Comment:
please remove all the unrelated formatting changes from this commit

Reply:
Done.